apps/qubit/modules/default/actions/alertsComponent.class.php

```diff
@@ -65,8 +65,11 @@ public function execute($request)
           );
           $alertType = $alertTypes[$job->getStatusId()];
 
-          // Add as conditional alert
+          // If job is complete, allow it to be deleted by the user
           $deleteUrl = $this->context->controller->genUrl('jobs/delete?id='. $job->id);
+          $deleteUrl = ($job->getStatusId() == QubitTerm::JOB_STATUS_COMPLETED_ID) ? $deleteUrl : null;
+
+          // Add as conditional alert
           array_push($this->conditionalAlerts, array('type' => $alertType, 'message' => $message, 'deleteUrl' => $deleteUrl));
```

Comment:
This is for controlling what happens when the X (to close a notification) is clicked. If the $deleteUrl is null then clicking X will have the standard behaviour of dismissing the notification.